jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -406,6 +406,54 @@
         </v-text-field>
       </v-row>
 
+      <v-row v-if="ext_type_selected === 'Horne'">
+        <v-select
+          attach
+          :menu-props="{ left: true }"
+          :items=" horne_ext_profile_items.map(i => i.label)"
+          v-model="horne_ext_profile_selected"
+          label="Profile Type"
+          hint="Profile to use for Horne extractoin."
+          persistent-hint
+        ></v-select>
+      </v-row>
+
+      <v-row v-if="horne_ext_profile_selected === 'Self (interpolated)'">
```

Comment:
these all either need `ext_type_selected === 'Horne' && ...` or the whole section could be wrapped in a `<div v-if="ext_type_selected == 'Horne'>`.